src/utils.js

```diff
@@ -210,12 +210,14 @@ var Utils = exports.Utils = {
    **/
   disablePrivacyBadgerForOrigin: function(origin){
     if(localStorage.disabledSites === undefined){
-      localStorage.disabledSites = JSON.stringify([origin]);
+      var disabledSites = new Object();
+      disabledSites[origin] = true;
+      localStorage.disabledSites = JSON.stringify(disabledSites);
```

Comment:
@cooperq  I am defining a new Object only when disabledSites is empty. When it already has some disabledSites it appends as you said.